doc/user/cluster_tls.md

```diff
@@ -85,3 +85,30 @@ $ kubectl create secret generic operator-etcd-client-tls --from-file=etcd-ca-crt
 ```
 
 Pass `operator-etcd-client-tls` to `operatorSecret` field.
+
+### Access secure cluster
+
+Since we have set this domain in SAN in etcd server's client cert:
+```
+example-client.default.svc.cluster.local
+```
+We are gonna use this to access etcd cluster.
+If you want to use different DNS name or IP, you need to add that when generating etcd server's client certs.
+
+We are reusing the same certs opeartor is using:
```

Comment:
I think some of this is good for testing but agree that users need to create their own.   Just a pain for quick evaluations.